modules/i18n/src/translations/si-LK/portals/console.ts

```diff
@@ -5093,7 +5093,7 @@ export const console: ConsoleNS = {
                         },
                         name: {
                             label: "නම",
-                            placeholder: "ගුණාංගය සඳහා නමක් ඇතුළත් කරන්න",
+                            placeholder: "සංදර්ශක නාමය ඇතුළත් කරන්න",
```

Comment:
Display Name is translated into `ප්රදර්ශන නාමය` in Google Translate, and I feel like that's more appropriate than using the term `සංදර්ශක නාමය`. WDYT :) ?

Reply:
https://github.com/wso2/identity-apps/pull/3888/commits/91cf27f7e6cb86b4d0e412d7ecac191e2a94f5a0